shinhye/models/userDao.js

```diff
@@ -0,0 +1,39 @@
+const { AppDataSource } = require("./index");
+
+const createUser = async (email, profileImage, password) => {
+  try {
+    await AppDataSource.query(
+      `INSERT INTO users(
+                  email,
+                  profile_image,
+                  password
+                  ) VALUES (?, ?, ?);
+              `,
+      [email, profileImage, password]
+    );
+  } catch (err) {
+    console.log(err);
+    throw error;
+  }
+};
+
+const checkEmail = async (email) => {
+  try {
+    const userInfo = await AppDataSource.query(
+      `SELECT
+      id,
+      email
```

Comment:
해당 부분도 들여쓰기를 수정하여 포맷을 일정하게 유지해주세요!

```suggestion
        id,
        email
```